src/controllers/dynakube/activegate/capability/capability_test.go

```diff
@@ -1,587 +1,86 @@
-// 						ServicePorts: AgServicePorts{
-// 							Webserver: true,
-// 							Statsd:    true,
-// 						},
-// 					},
-// 					initContainersTemplates: []v1.Container{
-// 						{
-// 							Name:            initContainerTemplateName,
-// 							ImagePullPolicy: v1.PullAlways,
-// 							WorkingDir:      k8scrt2jksWorkingDir,
-// 							Command:         []string{"/bin/bash"},
-// 							Args:            []string{"-c", k8scrt2jksPath},
-// 							VolumeMounts: []v1.VolumeMount{
-// 								{
-// 									ReadOnly:  false,
-// 									Name:      trustStoreVolume,
-// 									MountPath: activeGateSslPath,
-// 								},
-// 							},
-// 						},
-// 					},
-// 					containerVolumeMounts: []v1.VolumeMount{{
-// 						ReadOnly:  true,
-// 						Name:      trustStoreVolume,
-// 						MountPath: activeGateCacertsPath,
-// 						SubPath:   k8sCertificateFile,
-// 					}},
-// 					volumes: []v1.Volume{{
-// 						Name: trustStoreVolume,
-// 						VolumeSource: v1.VolumeSource{
-// 							EmptyDir: &v1.EmptyDirVolumeSource{},
-// 						},
-// 					}},
-// 				},
-// 			},
-// 		},
-// 	}
-// 	for _, tt := range tests {
-// 		t.Run(tt.name, func(t *testing.T) {
-// 			if got := NewMultiCapability(tt.args.dynakube); !reflect.DeepEqual(got, tt.want) {
-// 				t.Errorf("NewMultiCapability() = %v, want %v", got, tt.want)
-// 			}
-// 		})
-// 	}
-// }
+import (
+	"testing"
+
+	dynatracev1beta1 "github.com/Dynatrace/dynatrace-operator/src/api/v1beta1"
+	"github.com/stretchr/testify/assert"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	testNamespace     = "test-namespace"
+	testName          = "test-name"
+	testApiUrl        = "https://demo.dev.dynatracelabs.com/api"
+	expectedShortName = "activegate"
+	expectedArgName   = "MSGrouter,kubernetes_monitoring,metrics_ingest,restInterface,synthetic,beacon_forwarder,beacon_forwarder_synthetic"
+)
+
+func getTestInstance() *dynatracev1beta1.DynaKube {
+	return &dynatracev1beta1.DynaKube{
+		ObjectMeta: v1.ObjectMeta{
+			Namespace: testNamespace, Name: testName,
+		},
+		Spec: dynatracev1beta1.DynaKubeSpec{
+			APIURL: testApiUrl,
+			ActiveGate: dynatracev1beta1.ActiveGateSpec{
+				Capabilities: []dynatracev1beta1.CapabilityDisplayName{
+					dynatracev1beta1.RoutingCapability.DisplayName,
+					dynatracev1beta1.KubeMonCapability.DisplayName,
+					dynatracev1beta1.MetricsIngestCapability.DisplayName,
+					dynatracev1beta1.DynatraceApiCapability.DisplayName,
+					dynatracev1beta1.SyntheticCapability.DisplayName,
+				},
+			},
+		},
+	}
+}
+
+func getTestInstanceWithoutCapabilites() *dynatracev1beta1.DynaKube {
+	return &dynatracev1beta1.DynaKube{
+		ObjectMeta: v1.ObjectMeta{
+			Namespace: testNamespace, Name: testName,
+		},
+		Spec: dynatracev1beta1.DynaKubeSpec{
+			APIURL:     testApiUrl,
+			ActiveGate: dynatracev1beta1.ActiveGateSpec{},
+		},
+	}
+}
+
+func TestBuildProxySecretName(t *testing.T) {
+	t.Run(`test if build works`, func(t *testing.T) {
```

Comment:
```suggestion
	t.Run(`correct name`, func(t *testing.T) {
```
"test ..." is kind of redundant here.

Reply:
similar for other tests